src/stories/components/CustomTable/ItemCoreUnit/ItemCoreUnit.tsx

```diff
@@ -16,20 +16,20 @@ interface Props {
 export const ItemCoreUnit = ({ queryStrings, isLoading, columns, cu }: Props) => {
   const { isLight } = useThemeContext();
   return (
-    <Link href={`/core-unit/${cu?.shortCode}/${queryStrings}`} passHref>
-      <>
-        <TableWrapper>
+    <>
+      <TableWrapper>
+        <Link href={`/core-unit/${cu?.shortCode}/${queryStrings}`} passHref legacyBehavior>
```

Comment:
Please, use the `siteRoutes` instead of the path hardcoded

```ts
<Link href={`${siteRoutes.coreUnitAbout(cu?.shortCode)}/${queryStrings}`} passHref legacyBehavior>
```

Reply:
you are right